### PR TITLE
fix: create-tauri-app will fail when creating tauri.conf.json but src…

### PR DESCRIPTION
**Pull request by ikey4u**
…-taur directory is not exist

**What kind of change does this PR introduce?** (check at least one)

- [x] Bugfix
- [ ] Feature
- [ ] New Binding Issue #___
- [ ] Code style update
- [ ] Refactor
- [x] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes. Issue #___
- [x] No

**The PR fulfills these requirements:**

- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix: #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] A change file is added if any packages will require a version bump due to this PR per [the instructions in the readme](https://github.com/tauri-apps/tauri/blob/dev/.changes/readme.md).

If adding a **new feature**, the PR's description includes:
- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

### CODE REVIEW

**Review by nothingismagick — APPROVED** [review]
Great catch.

File: tooling/create-tauri-app/src/helpers/update-tauri-conf.ts [hunk 1]
```diff
@@ -2,15 +2,18 @@
...
 import { join } from 'path'
 import { TauriBuildConfig } from '../types/config'
 
 export function updateTauriConf(
   appDirectory: string,
   cfg: TauriBuildConfig
 ): void {
-  const tauriConfPath = join(appDirectory, 'src-tauri', 'tauri.conf.json')
+  const tauriConfDir = join(appDirectory, 'src-tauri')
+  mkdirSync(tauriConfDir, { recursive: true });
```
Comment (lucasfernog) [inline tooling/create-tauri-app/src/helpers/update-tauri-conf.ts:14]: I think this is not needed considering this function should only be called after `tauri init` execution finishes. Am I wrong @jbolda ?

File: tooling/create-tauri-app/src/helpers/update-tauri-conf.ts [hunk 2]
```diff
@@ -25,5 +28,6 @@ export function updateTauriConf(
     }
   }
 
+
```
Comment (nothingismagick) [inline tooling/create-tauri-app/src/helpers/update-tauri-conf.ts:31]: i think this extra line is why the eslint test failed.

File: tooling/create-tauri-app/src/helpers/update-tauri-conf.ts (hunk 1 above)
Comment (jbolda) [reply tooling/create-tauri-app/src/helpers/update-tauri-conf.ts:14]: That is my understanding. `tauri init` should be creating the folder and the `tauri.conf.json` file. @ikey4u did you run into an error that prompted this PR?

**Comment by lemarier** [comment]
Hi @ikey4u if you encounter an error because `tauri.conf.json` didn't exist, please open an issue we'll track down.

As @lucasfernog stated;
> I think this is not needed considering this function should only be called after `tauri init` execution finishes.

Thanks for your contribution we do appreciate it a lot.

Have a great day

**Pull request closed without merging**